src/status_im/contexts/wallet/bridge/input_amount/view.cljs

```diff
@@ -1,9 +1,9 @@
-(ns status-im.contexts.wallet.account.bridge.view
+(ns status-im.contexts.wallet.bridge.input-amount.view
```

Comment:
Shouldn't this be `bridge.select-asset`?

Reply:
yeah sounds good to me! will adjust!